src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -676,7 +676,7 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <comment>{0} should be the package identity of the package to verify</comment>
   </data>
   <data name="VerifyCommand_Failed" xml:space="preserve">
-    <value>Could not verify package(s) signature.</value>
+    <value>Package signature validation failed.</value>
```

Comment:
shouldn't it be `Package(s) ...` the verify command can accept more than one package as input

Reply:
This seems grammatically OK to me as is.